projects/storefrontapp/tsconfig.app.prod.json

```diff
@@ -8,6 +8,19 @@
       "@spartacus/vendor": ["dist/vendor"],
       "@spartacus/assets": ["dist/assets"],
       "@spartacus/my-account": ["dist/my-account"],
+      "@spartacus/my-account/organization": ["dist/my-account/organization"],
+      "@spartacus/my-account/organization/core": [
+        "dist/my-account/organization/core"
+      ],
+      "@spartacus/my-account/organization/occ": [
+        "dist/my-account/organization/occ"
+      ],
+      "@spartacus/my-account/organization/components": [
+        "dist/my-account/organization/components"
+      ],
+      "@spartacus/my-account/organization/assets": [
+        "dist/my-account/organization/assets"
+      ],
```

Comment:
Should we add all those secondary entry also in other tsconfig files ? @dunqan 
(in other files there is only `@spartacus/my-account/organization`)

Reply:
We have it synced everywhere now